lib/backends/sdc/containers.js

```diff
@@ -1507,15 +1507,28 @@ function buildVmPayload(opts, container, callback) {
         });
     }
     if (container.Labels) {
-        Object.keys(container.Labels).forEach(function _labelToTag(key) {
-            if (key === PACKAGE_SELECTION_LABEL) {
+        labelKeys = Object.keys(container.Labels);
+        for (i = 0; i < labelKeys.length; i++) {
+            labelKey = labelKeys[i];
+            if (labelKey === PACKAGE_SELECTION_LABEL) {
                 // PACKAGE_SELECTION_LABEL is a special label. It gets consumed
                 // here and doesn't end up in the user's list. Instead it's used
                 // in getPackage().
-                return;
+                continue;
             }
-            payload.tags[common.LABELTAG_PREFIX + key] = container.Labels[key];
-        });
+            if (triton_tags.isTritonTag(labelKey)) {
+                try {
+                    payload.tags[labelKey] = triton_tags.validateTritonTagStr(
+                        labelKey, container.Labels[labelKey]);
+                } catch (err) {
+                    callback(err);
```

Comment:
Yes definitely. Thanks. I'd forgotten to do that.

FWIW, I left triton-tags.js generic (i.e. not using the local errors.js) so that it could eventually be pulled out to a separate module (node-triton-tags.js or whatever) to have the validation shared between here and in VMAPI.